packages/libs/sdk-helpers/src/state.ts

```diff
@@ -73,7 +73,7 @@ export class State<T extends StateObject> {
       typeof newState === 'function' ? newState(this.#state) : newState;
 
     const nextState = { ...this.#state, ...internalNewState };
-    if (!this.#updateOnlyOnChange || !compareObjects(this.#state, nextState)) {
+    if (this.#forceUpdate || !compareObjects(this.#state, nextState)) {
```

Comment:
can you explain this change? (not the renaming, why the default is true now)

Reply:
`updateOnlyOnChange=false` is the same as `forceUpdate=true`